addon/data/shoot-panel.js

```diff
@@ -135,6 +141,17 @@ class ShootPanel extends React.Component {
     self.port.emit("copyLink", this.props.shot.viewUrl);
```

Comment:
I think the extra argument here is the bug, not below – below there is specific language about what is being copied to the clipboard, so if you passed a different link here that language would be incorrect.